genisys/modules/nat.py

```diff
@@ -67,4 +67,9 @@ def install_location(self: Self) -> Path:
 
     # end install_location
 
+    def setup_commands(self: Self) -> Union[List[str], List[List[str]]]:
+        return ["sudo netfilter-persistent save", "sudo netfilter-persistent reload", "sudo systemctl enable iptables", "sudo systemctl start iptables"]
```

Comment:
I think it's safe to assume the `genisys` application will be run as root when these commands are run (we may want to check for such when `install` is specified), so we can probably drop the `sudo`s.

Reply:
Changed it.